images/build/Dockerfiles/gitHubActions.Dockerfile

```diff
@@ -98,16 +98,29 @@ RUN ${IMAGES_DIR}/retry.sh "curl -o /usr/local/share/ca-certificates/verisign.cr
     
```

Comment:
You have probably seen the message from @horihiro at this point, and I imagine you'll adapt this for other images as well?

Reply:
Need to confirm with them if it only happened in GH action buster image.